ProceduralForest/Structures/model.h

```diff
@@ -0,0 +1,243 @@
+#ifndef MODEL_H
```

Comment:
File is never used and conflicts with current structure of the project

Reply:
it supposed to read the cube object as to wrap the skybox around it